client/my-sites/domains/index.js

```diff
@@ -12,7 +12,7 @@ import domainsController from './controller';
 import domainManagementController from './domain-management/controller';
 import SiftScience from 'lib/siftscience';
 import config from 'config';
-import paths from './paths';
+import * as paths from './paths';
```

Comment:
This looks like it doesn't belong to this PR, no?

Reply:
thanks! I removed it in 16a792b